instrumentation/kamon-system-metrics/src/main/scala/kamon/instrumentation/system/jvm/JvmMetrics.scala

```diff
@@ -137,19 +154,26 @@ object JvmMetrics {
       })
   }
 
+  class ClassLoadingInstruments(tags: TagSet) extends InstrumentGroup(tags) {
+    val loaded = register(ClassesLoaded, TagSet.from(Map("mode" -> "loaded")))
+    val unloaded = register(ClassesUnloaded, TagSet.from(Map("mode" -> "unloaded")))
+    val currentlyLoaded = register(ClassesCurrentlyLoaded, TagSet.from(Map("mode" -> "currently-loaded")))
```

Comment:
Those `mode` tags are not necessary anymore because each metric is a separate metric (as it should have been from the beginning of times!)